src/nexus/__init__.py

```diff
@@ -8,4 +8,4 @@
 from nexus import tools
 
 __version__ = "2.8.1.dev0"
-__all__ = ["NexusReader", "NexusWriter", "NexusFormatException", "handlers", "tools"]
+__all__ = ("NexusReader", "NexusWriter", "NexusFormatException", "handlers", "tools")
```

Comment:
What's the idea here? Conceptually, this **is** a list to me, not a tuple.

Reply:
oh weird. I'll revert this.